app/webpack/reactjs/components/art_modal.tsx

```diff
@@ -50,40 +45,58 @@ const ArtModalWindow: FC<ArtModalWindowProps> = ({ artPiece, handleClose }) => {
 
   useEventListener("keydown", keyDownHandler);
 
-  const handleNext: MouseEventHandler = (e) => {
-    e.preventDefault();
-    next();
-  };
-  const handlePrevious: MouseEventHandler = (e) => {
-    e.preventDefault();
-    previous();
-  };
-  const handleShowContactForm: MouseEventHandler = (e) => {
-    e.preventDefault();
-    showForm();
-  };
+  const handleNext: MouseEventHandler = useCallback(
```

Comment:
While we're here, let's leverage `useCallback` and `useMemo` so we prevent more re-rendering and think about performance.

Reply:
FWIW - we already did this on BSS.